lib/ansiblelint/rules/__init__.py

```diff
@@ -36,7 +36,10 @@ def verbose(self) -> str:
 
     @staticmethod
     def unjinja(text):
-        return re.sub(r"{{[^}]*}}", "JINJA_VAR", text)
+        text = re.sub(r"{{.+?}}", "JINJA_EXPRESSION", text)
```

Comment:
Is this the only place that is relevant to the PR?

Reply:
Yes.